providers/shared/references/Port/id.ftl

```diff
@@ -23,7 +23,18 @@
         },
         {
             "Names" : "Port",
-            "Type" : NUMBER_TYPE
+            "Children" : [
+                {
+                    "Names" : "Http",
+                    "Type" : NUMBER_TYPE,
+                    "Default" : 80
+                },
+                {
+                    "Names" : "Https",
+                    "Type" : NUMBER_TYPE,
+                    "Default" : 443
+                }
+            ]
```

Comment:
If you've added the protocol specific ports in the solution configuration it shouldn't be required in the reference data

Reply:
Yep, thanks. I had a misunderstanding on how these definitions and reference data worked that the meetings on Friday cleared up. I've fixed that in 389057e